### PR TITLE
Allow styleElement to update CSS Custom Properties with disableRuntime

### LINKED ISSUE #236: Allow `styleElement()` to update CSS Custom Properties when disableRuntime is set
External copy of [b/123235013](http://b/123235013)

### PR DESCRIPTION
**Pull request by azakus**
Fixes #236

### CODE REVIEW

File: src/scoping-shim.js [hunk 1]
```diff
@@ -259,17 +267,25 @@ export default class ScopingShim {
...
     } else {
       this.styleElementNativeVariables(host, styleInfo);
     }
   }
+  /**
+   * @param {StyleInfo} styleInfo
+   * @param {Object=} overrideProps
+   */
+  _mixOverrideStyleProps(styleInfo, overrideProps) {
+    if (!styleInfo || !overrideProps) {
```
Comment (sorvell) [inline src/scoping-shim.js]: Let's move these conditionals out of here. You should only need the `overrideProps` check at https://github.com/webcomponents/shadycss/pull/237/files#diff-a15f14a51481bba582abcc3f00fad9d0R270

File: tests/disable-runtime.html [hunk 2]
```diff
@@ -83,6 +83,15 @@
         const style = document.querySelector('custom-style > style');
         assert.notInclude(style.textContent, '--foo_-_border', 'custom style should not have ApplyShim applied');
       });
+      test('styleElement with props still works', function() {
+        makeElement('update-props');
+        const el = document.createElement('update-props');
+        document.body.appendChild(el);
+        assert.equal(window.ShadyCSS.getComputedStyleValue(el, '--bar'), '');
+        debugger;
```
Comment (sorvell) [inline tests/disable-runtime.html]: Remove.

File: src/scoping-shim.js (hunk 1 above)
Comment (azakus) [reply src/scoping-shim.js]: Done.

File: tests/disable-runtime.html (hunk 2 above)
Comment (azakus) [reply tests/disable-runtime.html]: Done.

**Pull request merged**